lib/screens/home/explorer/proposals_flow/proposal_vote_confirm/proposal_vote_confirm_screen.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/common/widgets/button.dart';
+import 'package:provenance_wallet/common/widgets/pw_app_bar.dart';
+import 'package:provenance_wallet/common/widgets/pw_list_divider.dart';
+import 'package:provenance_wallet/common/widgets/pw_slider.dart';
+import 'package:provenance_wallet/screens/home/transactions/details_item.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ProposalVoteConfirmScreen extends StatefulWidget {
+  const ProposalVoteConfirmScreen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _ProposalVoteConfirmScreen();
+}
+
+class _ProposalVoteConfirmScreen extends State<ProposalVoteConfirmScreen> {
+  double _gasEstimate = 1.25;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: PwAppBar(
+        leadingIcon: PwIcons.back,
+        title: "Vote Confirm",
+      ),
+      body: Container(
+        color: Theme.of(context).colorScheme.neutral750,
+        child: ListView(
+          children: [
+            DetailsItem(
+              title: "Proposer Address",
+              endChild: PwText(
+                "47",
+                overflow: TextOverflow.fade,
+                softWrap: false,
+                color: PwColor.neutralNeutral,
+                style: PwTextStyle.body,
+              ),
+            ),
+            PwListDivider(
+              indent: Spacing.largeX3,
+            ),
+            DetailsItem(
+              title: "Voter Address",
+              endChild: PwText(
+                "hj4k3h1jk43h1jklh4j4hkhk4h3kh4lk1hjk",
```

Comment:
Is this UI feature gated with a dev setting or something similar? We should keep the `develop` branch in a shippable state.

Reply:
This isn't going into `develop`, and there is no way to get to this screen yet - that's coming in the next pull request.

Reply:
Oops, missed that